### DISCUSSION TITLE
Is it possible to play open.spotify links with start_playback?

### DISCUSSION THREAD

**Discussion by timlag1305** [ramsayleung/rspotify#348] [General]
I was reading through the Spotify web API documentation and I think this a limitation on their side. However, I thought I'd ask here anyway. I would like to play songs via https://open.spotify.com links rather than URIs when calling `start_playback`. Is that possible? If not, is it possible to convert the links to URIs?

**Comment by ramsayleung** [comment]
Sorry for not getting your point, it seems what you want is nothing to do with `Rspotify`, right?

**Comment by timlag1305** [comment]
You're right, this is not directly related to Rspotify, but I am using this information in software that uses rspotify. I was looking at understanding how to convert a URL to a URI for the Spotify. In any case I'll close this. I figured it out anyway.
